internal/characters/yaemiko/charge.go

```diff
@@ -41,13 +40,21 @@ func (c *char) ChargeAttack(p map[string]int) action.ActionInfo {
 		windup = 14
 	}
 
-	// TODO: check snapshot delay
-	c.Core.QueueAttack(
-		ai,
-		combat.NewBoxHit(c.Core.Combat.Player(), c.Core.Combat.PrimaryTarget(), nil, 2, 2),
-		0,
-		chargeHitmark-windup,
-	)
+	// starts at recorded hitmark and +1.65m in target direction
+	// moves at 11m/s, one attack every 0.15s (9f), so it moves at 11 * 0.15 = 1.65m per attack
+	// gets gated by special damage sequence (once every 0.5s)
+	initialPos := c.Core.Combat.PrimaryTarget().Pos()
```

Comment:
Should this be offset from player position rather than starting on target position?

Reply:
player, will fix in a bit